src/microsAuthentication/content.md

```diff
@@ -0,0 +1,1118 @@
+If your infrastructure consists of several applications interacting with each other, you might have faced the 
+issue of securing the communication between services to prevent unauthenticated requests. Imagine having three apps:
+
+1. An API
+1. A data store
+1. A secret store
+
+You might want the secret store only to reply to requests to the API and reject any request from the data store.
+
+How would the secret store decide to authenticate or deny the request?
+
+A popular approach is to request and pass identity tokens to every call within services. So instead of 
```

Comment:
I know this sounds weird, but I have a single sentence per paragraph.

The content is spaced and easier to read.

Reply:
What if we have a linter rule to detect this?